ui/app/components/buy-button-subview.js

```diff
@@ -8,6 +8,11 @@ const ShapeshiftForm = require('./shapeshift-form')
 const Loading = require('./loading')
 const AccountPanel = require('./account-panel')
 const RadioList = require('./custom-radio-list')
+const networkNames = {
```

Comment:
maybe put this in app/scripts/config that way we can start reusing it else where.